modules/compliance-tailored-profiles.adoc

```diff
@@ -65,6 +65,9 @@ spec:
 |`disableRules`
 |A list of name and rationale pairs. Each name refers to a name of a rule object that is to be disabled. The rationale value is human-readable text describing why the rule is disabled.
 
+|`manualRules`
+| A list of name and rationale pairs. When a rule is added as a manual rule, it will always show manual as the check result status, and remediation will not be generated. This attribute is automatic and by default, has no values when set as a manual rule.
```

Comment:
The second sentence is a little wordy. Also when describing the check result status as output (like `manual`), put it in backticks:

"When a manual rule is added, the check result status will always be `manual` and remediation will not be generated."

Reply:
Done! Thanks so much for your help and review! :)